test/bdd/fixtures/auth-rest/hydra-config/thirdparty_hydra_configure.sh

```diff
@@ -19,4 +19,20 @@ hydra clients create \
     --callbacks https://localhost:8070/oauth2/callback
 # TODO it would be great to check the exit status of the hydra command
 #  https://github.com/trustbloc/auth/issues/67
-echo "Finished creating client!"
+echo "Finished creating oidc client!"
+
+echo "Creating gnap client for auth.rest.example.com..."
```

Comment:
We are creating OIDC client in both scenarios - probably `Creating oidc client for gnap flow auth.rest.example.com` more apt in this case